package.json

```diff
@@ -9,6 +9,7 @@
     "start": "webpack --watch --progress --mode=development",
     "start:profiles": "webpack-dev-server --progress --mode=development --config webpack.profiles.config.js --inline",
     "start:firefox": "webpack --watch --progress --mode=development --env.PLATFORM=firefox",
+    "run:firefox": "cd build/development/firefox && web-ext run",
```

Comment:
Prefer to run it via `start:firefox` in dev, since it uses webpack watch mode.

Reply:
It's a command that only runs the navigator with the extension installed. You think we don't need this command?

Reply:
Oh I see! This is actually interesting let's keep it.

Reply:
Will check if webpack watch mode works with this command